src/seed/seedSuperAdmin.js

```
@@ -1,26 +1,29 @@
-  superAdmin: { firstName, lastName, email, password }
-} = require('~/configs/config')
-const {
-  roles: { SUPERADMIN }
+  roles: { SUPERADMIN, ADMIN }
 } = require('~/consts/auth')
 const logger = require('~/logger/logger')
+const bcrypt = require('bcrypt');
 
 const SeedSuperAdmin = {
   createSuperAdmin: async () => {
+    const studentPassword = process.env.MAIL_PASS;
```

ESLint

[error] Extra semicolon.

(semi)